debian/jessie/foreman-installer/control

```diff
@@ -2,7 +2,7 @@ Source: foreman-installer
 Section: ruby
 Priority: optional
 Maintainer: Greg Sutcliffe <gsutclif@redhat.com>
-Build-Depends: debhelper (>= 7.0.50~), asciidoc, rake, libxml2-utils,
+Build-Depends: debhelper (>= 7.0.50~), asciidoc, rake, libxml2-utils, ruby-kafo (>= 0.7.1),
```

Comment:
Oh, that's why it changed.  The package (well, the gem itself)'s kafo-export-params is broken.  Adding it to the build root means it tries to export parameters for the man page, exposing the problem.

I think you can safely leave this out of build-deps for now and it'll build as it used to do, without the known parameters listed in the man page.  We'll have to fix kafo-export-params and add it again later.